llama_stack/distribution/store/registry.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+import json
+
+from typing import Protocol
+
+import pydantic
+
+from llama_stack.distribution.datatypes import RoutableObjectWithProvider
+
+from llama_stack.providers.utils.kvstore import KVStore
+
+
+class Registry(Protocol):
+    async def get(self, identifier: str) -> [RoutableObjectWithProvider]: ...
+    async def register(self, obj: RoutableObjectWithProvider) -> None: ...
+
+
+KEY_FORMAT = "distributions:registry:{}"
+
+
+class DiskRegistry(Registry):
+    def __init__(self, kvstore: KVStore):
+        self.kvstore = kvstore
+
+    async def get(self, identifier: str) -> [RoutableObjectWithProvider]:
+        # Get JSON string from kvstore
+        json_str = await self.kvstore.get(KEY_FORMAT.format(identifier))
+        if not json_str:
+            return []
+
+        # Parse JSON string into list of objects
```

Comment:
nit: remove the comments it is pretty self explanatory code

Reply:
done.